src/FixerConfiguration/FixerOptionBuilder.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\FixerConfiguration;
+
+final class FixerOptionBuilder
```

Comment:
They can, just as we can too. But the builder is more convenient to avoid passing all the arguments to `FixerOption` constructor.